test/MarketMaker.js

```diff
@@ -20,28 +20,21 @@ contract('marketMaker', function(accounts) {
 
   var mm;
   var token;
+  var crowdsale;
```

Comment:
let's avoid declaring this globally, do we need it? we can just add const every time we initialize it in the rest of the test

Reply:
:+1: